app/assets/javascripts/spree/return_authorization.js

```diff
@@ -1,2 +1,5 @@
 //= require 'spree/backend/returns/return_item_selection'
 //= require 'spree/frontend/return_item_options_selection'
+//= require jquery
+//= require jquery_ujs
```

Comment:
Why require jquery_ujs?

Reply:
Hello,

Could you please merge this pull request?